File: tests/python/exceptions/nested_try.py
# vybe-test: python/exceptions/nested_try
# origin: languages/python/tests/python/test_exceptions.rs

try:
    try:
        x = 1 / 0
    except ZeroDivisionError:
        print("inner")
        raise ValueError("converted")
except ValueError:
    print("outer")

File: tests/php/error_handling/nested_try.php
<?php
// vybe-test: php/error_handling/nested_try
// origin: languages/php/tests/php/test_error_handling.rs
// vybe-test-mode: compile

try { try { throw new Exception('inner'); } catch (Exception $e) { throw new Exception('rethrow'); } } catch (Exception $e) { echo 'outer'; }
